polygeist/lib/Dialect/Polygeist/Transforms/LoopInternalization.cpp

```diff
@@ -44,6 +45,111 @@ using namespace mlir::polygeist;
 
 namespace {
 
+using ValueOrUnsignedTy = std::variant<Value, unsigned>;
```

Comment:
I don't see the point here to have a using declaration.

Reply:
removed.